speakeasy/windows/winemu.py

```diff
@@ -634,6 +636,30 @@ def init_teb(self, thread, peb):
         elif self.get_arch() == _arch.ARCH_AMD64:
             thread.init_teb(self.gs_addr, peb.address)
 
+    def init_tls(self, thread):
+        """
+        Initialize implicit thread local storage. Meant to be
+        called after init_teb.
+        """
+        ptrsz = self.get_ptr_size()
+        run = self.curr_run
+        module = self.get_mod_from_addr(run.start_addr)
+
+        if module:
+            modname = module.emu_path
+            modname = modname[modname.find("\\")+2:]
```

Comment:
https://docs.python.org/3/library/os.path.html#os.path.basename

Reply:
That will not work on my macOS machine because all directories for emulation are windows directories, backslash vs. forward slash, but I can change this to use string's split